containers/azure-functions-pwsh-7/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# Find the Dockerfile for mcr.microsoft.com/azure-functions/powershell:3.0-powershell7-core-tools at this URL
+# https://github.com/Azure/azure-functions-docker/blob/master/host/3.0/buster/amd64/powershell/powershell7-core-tools.Dockerfile
+
+FROM mcr.microsoft.com/azure-functions/powershell:3.0-powershell7-core-tools
```

Comment:
gosh I wish that you could just put the image and tag in the `devcontainer.json` somehow... this `Dockerfile` is kinda silly.

@Chuxel is that possible?

Reply:
Yes you can do it. Just use the "image" tag instead of "Dockerfile". We tend not to since it doesn't give a place for the customer to add their own software is the challenge.  You'd may want to provide the link to the full Dockerfile for those that want to customize.